src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] hashTable;
+
+    public MyHashMap() {
+        hashTable = new Node[INITIAL_CAPACITY];
```

Comment:
Use the diamond operator to avoid raw type usage. It should be 'new Node<>[INITIAL_CAPACITY]' to specify that you are creating an array for a generic Node class without explicit type parameters.